apps/discord/discord_win.py

```diff
@@ -90,8 +90,29 @@ def discord_answer_call():
     def discord_decline_call():
         actions.key("esc")
 
+    def discord_start_call():
+        actions.key("ctrl-'")
+
     def discord_go_current_call():
         actions.key("ctrl-shift-alt-v")
 
     def discord_toggle_dms():
         actions.key("ctrl-alt-right")
+
+    def discord_create_join_server():
+        actions.key("ctrl-shift-n")
+
+    def discord_toggle_soundboard():
+        actions.key("ctrl-shift-b")
+
+    def discord_create_private_group():
+        actions.key("ctrl-shift-t")
+
+    def discord_channel_oldest_unread():
+        actions.key("shift-pageup")
+
+    def discord_open_support():
```

Comment:
Wondering if this should be a more general command to open help rather than something discord specific.

Reply:
Personally, I'm for it being a general command. However, I'm unsure as to what a clean syntax for that would look like. Help is already taken. It could be `app help` maybe?

Is there a standard way of handling common functionalities between applications that need to be differentiated from talon ones?

Reply:
Sure, you could consider defining a tag for it, or (probably better in this case) implementing it for an entire platform at a time via an action.